src/base/bittorrent/torrentinfo.cpp

```diff
@@ -309,7 +322,11 @@ QVector<QByteArray> TorrentInfo::pieceHashes() const
     hashes.reserve(count);
 
     for (int i = 0; i < count; ++i)
+#if LIBTORRENT_VERSION_NUM < 10200
         hashes += { m_nativeInfo->hash_for_piece_ptr(i), libtorrent::sha1_hash::size };
+#else
+        hashes += { m_nativeInfo->hash_for_piece_ptr(makePieceIndex(i)), libtorrent::sha1_hash::size() };
```

Comment:
I think you can use this line for both version, right? `makePieceIndex()` already provides the abstraction

Reply:
Notice `libtorrent::sha1_hash::size`became a function